Dziedziczenie.sln

```diff
@@ -3,7 +3,9 @@ Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio Version 16
 VisualStudioVersion = 16.0.29806.167
 MinimumVisualStudioVersion = 10.0.40219.1
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Dziedziczenie", "Dziedziczenie\Dziedziczenie.csproj", "{247DF9EA-7B57-4188-AD27-92E36FF605F6}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Dziedziczenie", "Dziedziczenie\Dziedziczenie.csproj", "{247DF9EA-7B57-4188-AD27-92E36FF605F6}"
```

Comment:
Prosiłbym jeszcze żeby przywrócił Pan moje ID projektu/solucji. To może zepsuć pracę innym studentom.